compiler/testData/codegen/box/invokedynamic/sam/inlineOnly.kt

```diff
@@ -0,0 +1,21 @@
+// TARGET_BACKEND: JVM
+// JVM_TARGET: 1.8
+// SAM_CONVERSIONS: INDY
+// WITH_RUNTIME
+
+// CHECK_BYTECODE_TEXT
+// JVM_IR_TEMPLATES
+// 0 java/lang/invoke/LambdaMetafactory
```

Comment:
Did this actually use invokedynamic without your changes? The original issue used a Java interface I can't reproduce the problem with `fun interface`...

Reply:
No, you are right. I simplified it at the end and I shouldn't have. :-\ I'll upload a change to just use `java.util.function.Consumer` which this test did when I fixed the issue. :)